packages/components/src/FieldDateRange/FieldDateRange.story.tsx

```diff
@@ -24,69 +24,45 @@
 
  */
 import React, { FC, useState } from 'react'
-import {
-  Button,
-  FieldSelect,
-  Paragraph,
-  Popover,
-  PopoverContent,
-} from '@looker/components'
-import { DateFormat } from '@looker/components/src/DateFormat'
-import { FieldDateRange } from '@looker/components/src/FieldDateRange'
-import { InputDateRange } from '@looker/components/src/InputDateRange'
-import { Locales } from '@looker/components/src/utils/i18n'
+import { Story } from '@storybook/react/types-6-0'
+import { Locales } from '../utils/i18n'
+import { FieldSelect } from '../Form/Fields/FieldSelect'
+import { Paragraph } from '../Text/Paragraph'
+import { DateFormat } from '../DateFormat'
+import { FieldDateRange, FieldInputDateRangeProps } from '.'
 
 interface DateRange {
   from?: Date
   to?: Date
 }
 
 export default {
-  title: 'Forms/DateRange',
+  title: 'Forms/FieldDateRange',
```

Comment:
`component` key enables Storybook Docs feature to show props for the component properly.

On the title - we've been flattening these for ease-of-embedding within Gatsby in upcoming docs revisions.

```suggestion
  component: FieldDateRange,
  title: 'FieldDateRange',
```